content/applications/inventory_and_mrp/inventory/product_management/product_replenishment/reordering_rules.rst

```diff
@@ -79,43 +83,187 @@ Reordering Rules`, then click :guilabel:`Create`, and fill out the new line as f
+falls below a specified minimum. Products on this dashboard are called *needs*, because they are
+needed to fulfill upcoming sales orders, but the forecasted quantity is not enough.
 
-Set a preferred route for reordering
-====================================
+The replenishment dashboard, accessible by navigating to :menuselection:`Inventory app -->
+Operations --> Replenishment`, considers sales order deadlines, forecasted stock levels, and vendor
+lead times. It displays needs **only** when it is time to reorder items.
+
+.. note::
+   If the one-day window for ordering products is too short, skip to the :ref:`visibility days
+   <inventory/product_management/visibility-days>` section to make the need appear on the
+   replenishment dashboard a specified number of days in advance.
+
+When a product appears on the replenishment dashboard, clicking the :guilabel:`Order Once` button
+generates the purchase or manufacturing order with the specified amounts :guilabel:`To Order`.
+
+.. image:: reordering_rules/manual.png
+   :align: center
+   :alt: Click the Order Once button on the replenishment dashboard to replenish stock.
+
+.. _inventory/product_management/visibility-days:
+
+Visibility days
+===============
+
+.. note::
+   Ensure :doc:`lead times <../../shipping_receiving/advanced_operations_shipping/scheduled_dates>`
+   are understood before proceeding with this section.
+
+When :ref:`manual reordering rules <inventory/product_management/manual-rr>` are assigned to a
+product, *visibility days* make the product appear on the replenishment dashboard
+(:menuselection:`Inventory app --> Operations --> Replenishment`) a certain number of days in
+advance.
+
+.. example::
+   A product has a manual reordering rule set to trigger when the stock level falls below four
+   units. The current on-hand quantity is ten units.
+
+   The current date is February twentieth, and the *delivery date* on a sales order (in the
+   :guilabel:`Other Info` tab) is March third — twelve days from the current date.
+
+   The :ref:`vendor lead time <inventory/management/purchase-lt>` is four days, and the
+   :ref:`purchase security lead time <inventory/management/purchase-security-lt>` is one day.
+
+   When the :guilabel:`Visibility Days` field of the reordering rule is set to zero, the product
+   appears on the replenishment dashboard five days before the delivery date, which, in this case,
+   is February twenty-seventh.
+
+   .. image:: reordering_rules/need-dates.png
+      :align: center
+      :alt: Graphic representing when the need appears on the replenishment dashboard: Feb 27th.
+
+   To see the product on the replenishment dashboard for the current date, February twentieth, set
+   the :guilabel:`Visibility Days` to `7.00`.
+
+To determine the amount of visibility days needed to see a product on the replenishment dashboard,
+subtract **today's date** from the **date the need appears** on the replenishment dashboard.
```

Comment:
I fully see the value of bolding these elements in the sentence to help illustrate the subtraction that needs to take place, but I'm curious if the same could be accomplished with a couple of "format/style guide-friendly" asterisks. Not a requirement, just a suggestion. Totally your call.